code/include/swoc/swoc_ip.h

```diff
@@ -187,6 +187,8 @@ union IPEndpoint {
   static string_view family_name(sa_family_t family);
 };
 
+constexpr auto const _BITSPERBYTE = std::numeric_limits<unsigned char>::digits;
```

Comment:
Use of reserved name. I think you remove this and do a global replace to fix it.

Reply:
Oh, good to know.